modules/ocs2/src/main/scala/gem/ocs2/SmartGcalImporter.scala

```diff
@@ -175,35 +174,40 @@ object SmartGcalImporter extends TaskApp with DoobieClient {
                     parser:         KeyParser[K],
                     unindexer:      ConnectionIO[Int],
                     writer:         SmartGcalWriter[K],
-                    indexer:        ConnectionIO[Int]): Task[Unit] = {
+                    indexer:        ConnectionIO[Int]): IO[Unit] = {
 
-    def lines(l: GcalLampType): Process[Task, SmartGcalLine[K]] =
-      scalaz.stream.io
-          .linesR(new File(dir, fileName(instFilePrefix, l)).getPath)
+    def lines(l: GcalLampType): Stream[IO, SmartGcalLine[K]] =
+      io.file.readAll[IO](new File(dir, fileName(instFilePrefix, l)).toPath, 4096)
+          .through(text.utf8Decode)
+          .through(text.lines)
           .map(_.split(',').map(_.trim).toList)
           .map(parseLine(_, l, parser))
```

Comment:
Ok, shall I add that line and then call it good for now? It would be nice to add some tests for the importer but we need scrubbed data or some kind of encrypted archive.